akka-cluster/src/main/resources/reference.conf

```diff
@@ -34,7 +34,14 @@ akka {
     # in case of a network partition the persistent actors in the non-surviving partitions
     # must be stopped before corresponding persistent actors are started somewhere else.
     down-removal-margin = 20s
-    
+
+    # By default, the leader will not move 'Joining' members to 'Up' during a network
+    # split. This feature allows the leader to accept 'Joining' members to be 'WeaklyUp'
+    # so they become part of the cluster even during a network split. The leader will
```

Comment:
This only explains the "upsides" of this feature and the weaklyup state. If I only read this I would be to tempted to turn this on. We should explain here in one simple sentence that this state has various limitations compared to be fully Up.